Helper/Data.php

```diff
@@ -91,6 +94,8 @@ public function getConfigData($field, $store = null)
      * @param string $str
      * @return string
      */
+	 
+	 
     public function filter($str)
     {

```

Comment:
It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.